integrationtests/tests/fixtures/mod.rs

```diff
@@ -1026,6 +1032,12 @@ impl FederationTest {
             let mut override_modules = override_modules(cfg.clone(), db.clone()).await;
 
             let mut modules = BTreeMap::new();
+            // We currently have no way to enforce that modules are not reading
```

Comment:
I... recognize this code. If you want to re-use it, probably better to put original one in some method you can call here that returns that btreemap.

Reply:
:) Fixed